readme.md

```diff
@@ -46,7 +46,7 @@ https://developers.google.com/google-apps/calendar/quickstart/nodejs
 ## Usage instructions ##
 
 ```node
-node ./cli.js --calID 'yourGoogleCalenderID' --meetupApiKey 'yourMeetupApiKey'
+node meetsy --calID 'yourGoogleCalenderID' --meetupApiKey 'yourMeetupApiKey' --secret './googleApi_clientSecret.json'
```

Comment:
`node meetsy` => `meetsy ...`. Also you should change the installation instruction from "clone repo" to "npm install -g meetsy".